### PR TITLE
Port several foods and food sprites

### PR DESCRIPTION
**Pull request by Arokha**
# Sprite Replacement
Replaces all the bread and sandwich sprites with better ones from /tg/ (see icondiffbot), and ports approximately a trillion donut types. Someone like @Ketrai or @MarinaGryphon will have to make all the recipes.

image

# TGMC MREs
Ported the TGMC MREs as an alternative to our existing ones (didn't replace anything). They work like the TGMC MREs and generate less trash as a result, though they still generate one piece for the MRE pack itself when you remove the last item (as is on TGMC).

# Smol Foods
TGMC has all these smol foods because I guess they want realistic sizes, but I thought it'd be fun to stock 3 smol wrapped ones in a vendor without much food value because they look like something terrible you'd find in a vending machine.

zWShOqXCs0
dreamseeker_Jc03iGdY1T

### CODE REVIEW

**Review by Leshana — APPROVED** [review]

**Comment by Very-Soft** [comment]
Me like!!

**Comment by Ketrai** [comment]
These donuts claim to be made fresh daily in a boutique bakery in New Reykjavik and delivered to Nanotrasen's hardworking asset protection crew. They're probably synthesized.

**Comment by Heroman3003** [comment]
Look good but there's something very off with animation for rotten meat sprite it seems? Might just be me/git weirdness.

**Comment by Arokha** [comment]
> Look good but there's something very off with animation for rotten meat sprite it seems? Might just be me/git weirdness.

That's just icondiffbot being weird. It's got flies flying around on it. It looks fine in reality, just IDB seems to have uh, some issues drawing it.

File: code/modules/food/food/snacks.dm [hunk 1]
```diff
@@ -384,43 +411,256 @@
...
+/obj/item/weapon/reagent_containers/food/snacks/donut/choc_sprinkles/jelly/Initialize()
 	. = ..()
 	reagents.add_reagent("nutriment", 3, nutriment_desc)
 	reagents.add_reagent("sprinkles", 1)
-	if(prob(30))
-		src.icon_state = "donut2"
-		src.overlay_state = "box-donut2"
-		src.name = "frosted donut"
-		reagents.add_reagent("sprinkles", 2)
-		center_of_mass = list("x"=19, "y"=16)
```
Comment (Ketrai) [inline code/modules/food/food/snacks.dm:409]: Why was this chance at a frosted donut removed? This isn't documented and essentially reduces frosted donuts to the realm of security donut box RNG and donut vending machines. Chefs should be able to make these, and this implementation is kinda fun. Sprinkles are also only obtainable this way, and required for certain recipes.

Comment (MarinaGryphon) [reply code/modules/food/food/snacks.dm:409]: You should be able to get sprinkles another way, and specifically make donuts with sprinkles. Rng shapes for chicken nuggets is aggravating enough as it is.

Comment (Heroman3003) [reply code/modules/food/food/snacks.dm:409]: As said in PR, part of the point is that A) a new recipe for sprinkles should be made down the line, not be a random RNG donut variety and B) all new donuts should have proper specific recipies of their own, not be entirely RNG. It literally says so in description.

**Pull request merged**